src/components/contactItem/contactItem.jsx

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 import Button from '../common/button';
 
 export default class ContactItem extends React.Component {
   deleteClickHandler = () => {
-    const { id, onDeleteClick } = this.props;
+    const { contact: { id, onDeleteClick } } = this.props;
     onDeleteClick(id);
   }
 
+  editClickHandler = () => {
```

Comment:
`handleSaveContact`

Reply:
This is not a save functionality, but edit